packages/esm-patient-registration-app/src/patient-registration/field/address/address-hierarchy.component.tsx

```diff
@@ -78,17 +85,27 @@ export const AddressHierarchy: React.FC = () => {
           width: '50%',
           paddingBottom: '5%',
         }}>
-        {addressLayout.map((attributes, index) => (
-          <ComboInput
-            key={`combo_input_${index}`}
-            name={attributes.name}
-            labelText={attributes.labelText}
-            id={attributes.name}
-            placeholder={attributes.labelText}
-            setSelectedValue={setSelectedValue}
-            selected={selected}
-          />
-        ))}
+        {useAddressHeirarchy
+          ? addressLayout.map((attributes, index) => (
+              <ComboInput
+                key={`combo_input_${index}`}
+                name={attributes.name}
+                labelText={t(attributes.name)}
+                id={attributes.name}
+                setSelectedValue={setSelectedValue}
+                selected={selected}
+              />
+            ))
+          : addressLayout.map((attributes, index) => (
```

Comment:
I'm really weirded out that this is using the same variable `addressLayout` for the dynamic one as the static version... Is this really the case?

Reply:
The address layout provides the fields to be shown on the patient registration page. Hence we are using the same address layout variable/